OasisDEX/oasis-borrow
# Show Fee Split inline (Ajna Multiply)

<img width="539" alt="Screenshot 2023-07-25 at 12 53 22" src="https://github.com/OasisDEX/oasis-borrow/assets/5372463/50ee7cd9-d080-4c60-88d1-a55402379442">

  
## Changes 👷‍♀️
- Show t/x fee and gas fee as separate items in transaction fee row in order info
- Hide oasis fee portion when zero (for example earn multiply)
- Hide dropdown when oasis fee portion is zero
  
## How to test 🧪
- Set up Ajna multiply position
- Observe whether t/x is broken apart
